src/components/cursor-controller.js

```diff
@@ -99,6 +109,11 @@ AFRAME.registerComponent("cursor-controller", {
   },
 
   pause: function() {
+    if (!this.inVR && this.isMobile && !this.hasPointingDevice) {
```

Comment:
I think its safe to just always remove these listeners and not do this check. Doing this check might open us up to lingering eventListeners in the case that you happen to enterVR at some point later, then this pause is called.